game_patch/rf/multi.h

```diff
@@ -151,6 +165,10 @@ namespace rf
     static auto& multi_tdm_get_red_team_score = addr_as_ref<int()>(0x004828F0); // returns ubyte in vanilla game
     static auto& multi_tdm_get_blue_team_score = addr_as_ref<int()>(0x00482900); // returns ubyte in vanilla game
     static auto& multi_num_players = addr_as_ref<int()>(0x00484830);
+    [[nodiscard]] inline int32_t multi_num_spawned_players() {
```

Comment:
I would prefer a name `multi_num_alive_players`, because alive is opposite to dead

Reply:
True but no one speaks like that. It is not idiomatic.

Reply:
"spawned" feels more ambiguous, like is dying player a spawned one? Not sure if being idiomatic in normal speech is important for naming functions. It's more important to make them non ambiguous, and clear